cli/pkg/release/http.go

```diff
@@ -141,14 +145,14 @@ func issueHttpRequest(req http.Request) error {
 	}
 	defer resp.Body.Close()
 
+	body, err := io.ReadAll(resp.Body)
+	strBody := string(body)
 	if resp.StatusCode != http.StatusCreated && resp.StatusCode != http.StatusOK {
-		body, err := io.ReadAll(resp.Body)
 		if err != nil {
 			return fmt.Errorf("response was not OK or Accepted\nresponse code: %v\nresponse body could not be be read, error: %w", resp.StatusCode, err)
 		}
-		strBody := string(body)
 		return fmt.Errorf("response was not OK or Accepted\nresponse code: %v\nresponse body:\n   %v", resp.StatusCode, strBody)
 	}
-
+	fmt.Println(strBody)
```

Comment:
is this println here on purpose?

Reply:
This was added to print out the response of the release creation. Right now the CLI doesn't print anything even though the release might have not been created because it might have already existed.
I also added it so that the deprecation warning of the release endpoint is printed out.